src/client.ts

```diff
@@ -1034,7 +1034,7 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
     contentType?: string,
     user?: UserResponse<StreamChatGenerics>,
   ) {
-    const data = addFileToFormData(uri, name, contentType);
+    const data = addFileToFormData(uri, name, contentType || 'multipart/form-data');
```

Comment:
We already have this default value within react and RN SDK. But adding the same to js client, for the cases where customer uses sendFile endpoint directly.